app/src/main/java/com/mupper/gobus/commons/scope/ScopedViewModel.kt

```diff
@@ -8,12 +8,12 @@ abstract class ScopedViewModel(uiDispatcher: CoroutineDispatcher) : ViewModel(),
     Scope by Scope.Impl(uiDispatcher) {
 
     init {
-        initScope()
+        this.initScope()
```

Comment:
Why is it neccesary to use "this" in this case?

Reply:
![image](https://user-images.githubusercontent.com/9106910/74979881-557b1500-53e4-11ea-82fb-f202671412db.png)